lib/tasks/routes.rake

```diff
@@ -3,10 +3,11 @@ unless defined?(Rails)
   task routes: :environment do
     require './config/application.rb'
     ApplicationEndpoint.routes.each do |api|
+      version = api.route_version.ljust(20)
       method = api.route_method.ljust(10)
       path = api.route_path.ljust(40)
       description = api.route_description
-      puts "     #{method} #{path} # #{description}"
+      puts "#{version} #{method} #{path} # #{description}"
```

Comment:
I snuck in an update so that the `rake routes` task describes versioned api routes more explicitly:

![image](https://cloud.githubusercontent.com/assets/197336/11946511/722c6d54-a829-11e5-995e-0dd78d02463b.png)